extensions/amp-story-player/validator-amp-story-player.protoascii

```diff
@@ -77,4 +77,40 @@ tags: {
     value_regex: "data:image\\/svg\\+xml;charset=utf-8,\\s*<svg height=\"\\d+(\\.\\d+)?\" width=\"\\d+(\\.\\d+)?\" xmlns=\"http:\\/\\/www\\.w3\\.org\\/2000\\/svg\" version=\"1\\.1\"\\/>|data:image\\/svg\\+xml;charset=utf-8,\\s*<svg height='\\d+(\\.\\d+)?\' width='\\d+(\\.\\d+)?\' xmlns='http:\\/\\/www\\.w3\\.org\\/2000\\/svg' version='1\\.1'\\/>|data:image\\/svg\\+xml;base64,[a-zA-Z0-9+\\/=]+"
     mandatory: true
   }
+  tag: "IMG"
+}
+tags: {
+  html_format: AMP
+  tag_name: "IMG"
+  spec_name: "AMP-STORY-PLAYER IMG"
```

Comment:
typically we use a `>` to show parent / child relationships in the names. so I'd encourage the usage of this as `spec_name: "amp-story-player > img"`. Also only the `tag_name` needs to be uppercase but since uppercase is used elsewhere in this file no objection, just an fyi.